users/models.py

```diff
@@ -4,6 +4,3 @@
 
 class User(AbstractUser):
     avatar = models.ImageField(default='default.jpg', upload_to='images/')
```

Comment:
avatar should be not required. You do not need a default value for the image. If you want the default image it should be on UI IN THIS CASE. Because it is more question of design. And now we have one design with one image, tomorrow we have another design with another image. And you will need to create migration and change for all your users this default image.